server/src/main/java/com/tencent/rss/server/ShuffleTaskManager.java

```diff
@@ -268,6 +268,9 @@ public long requireBuffer(int requireSize) {
       requireBufferIds.put(requireId,
           new PreAllocatedBufferInfo(requireId, System.currentTimeMillis(), requireSize));
     }
+    if (requireId == -1) {
+      ShuffleServerMetrics.counterTotalRequireBufferFailed.inc();
```

Comment:
how about move it to ShuffleServerGrpcService?

Reply:
Done